src/components/FilesRender.tsx

```
@@ -105,6 +113,36 @@ function Files({
 	const router = useRouter();
 	const [selectedFiles, setSelectedFiles] = useState<typeof files>([]);
 
+	// eslint-disable-next-line react-hooks/exhaustive-deps
+	const stableUser = useMemo(() => user, [user?.name]);
+	// eslint-disable-next-line react-hooks/exhaustive-deps
+	const stableClient = useMemo(() => client, [client?.connected]);
+
+	const sortedFileKey = (Array.isArray(files) ? files?.map((file) => file.id) : []).join('-');
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)